### ISSUE TITLE
build fails with serenity 1.5.1

### ISSUE THREAD

**Issue by ltalirz** [qcscine/serenity_wrapper#1]
Hi @nabbelbabbel 

I notice serenity 1.5.1 release included some changes to the `OrbitalController` class
https://github.com/qcserenity/serenity/blame/6c1721895c1e6cfe2036b3fb577dd042d7cdd06b/src/data/OrbitalController.cpp

It appears this breaks the serenity wrapper; I'm getting

``` 
_Args = {std::shared_ptr<Serenity::BasisController>}]' /mnt/nvme/root/spack-stage/spack-stage-scine-serenity-1.0.1-o7blkvu4eyfcmow77u46yl23plgqughh/spack-src/src/Serenity/Serenity/Calculators/CalculatorBase.cpp:209:68:  required from here
  >> 143    /include/c++/11.3.0/ext/new_allocator.h:162:11: error: no matching function for call to 'Serenity::OrbitalController<Serenity::Options::SCF_MODES::RESTRICTED>::OrbitalController(std::shared_ptr<Serenity::BasisController>)' 
```

**Comment by nabbelbabbel** [comment]
Indeed, the current version does not support the new Serenity release yet.

The next version of the wrapper will bump the wrapped Serenity version to 1.5.1.

**Comment by weymutht** [comment]
Version 3.0.0 of the Serenity wrapper, released today, support Serenity version 1.5.2. Therefore, I take the liberty of closing this issue.
